### PR TITLE
Garbage Cans are now functional

### LINKED ISSUE #8159: Garbage cans are marginally functional, don't accept items on click
**Problem Description**:
Garbage cans require you to climb on top of them to put trash in.

**What did you expect to happen**: 
For open garbage cans to accept items the way an open crate or disposal bin would.

**What happened instead**:
Trash in hand doesn't go in. Thrown items bounce off. Items click-dragged from the ground display a "x stuffs y in the garbage can" message and still don't go in.

**Why is this bad/What are the consequences:**
Results in mostly useless garbage cans cluttering up the place in front of the kitchen, at the mining outpost and wherever else they're mapped in.

**Steps to reproduce the problem**:
Try to put garbage in a garbage can without climbing on top of it.

**Possibly related stuff (which gamemode was it? What were you doing at the time? Was
anything else out of the ordinary happening?)**: Of course it's possible I missed something obvious here but I couldn't get an answer out of mhelp.

### PR DESCRIPTION
**Pull request by TullyBurnalot**
Fixes #8159

:cl:
fix: Can now place garbage in garbage cans without standing on top of them.
/:cl:

### CODE REVIEW

File: code/game/objects/structures/crates_lockers/crates.dm [hunk 1]
```diff
@@ -469,6 +469,15 @@
 				user.visible_message("[user] has secured [src]'s floor casters.", \
 						 								"<span class='notice'>You have secured [src]'s floor casters.</span>")
 				anchored = TRUE
+	if(opened)
+		if(isrobot(user))
+			return
+		if(!user.drop_item()) //couldn't drop the item
+			to_chat(user, "<span class='notice'>\The [W] is stuck to your hand, you cannot put it in \the [src]!</span>")
```
Comment (FalseIncarnate) [inline code/game/objects/structures/crates_lockers/crates.dm]: Remove both `'\the`, this is auto-appended by direct item references (`[W]` and `[src]`)

File: code/game/objects/structures/crates_lockers/crates.dm [hunk 2]
```diff
@@ -469,6 +469,15 @@
 				user.visible_message("[user] has secured [src]'s floor casters.", \
 						 								"<span class='notice'>You have secured [src]'s floor casters.</span>")
 				anchored = TRUE
+	if(opened)
```
Comment (Fox-McCloud) [inline code/game/objects/structures/crates_lockers/crates.dm]: why not just do 
```
	else
		..()
```

instead of having all of this?

Comment (FalseIncarnate) [reply code/game/objects/structures/crates_lockers/crates.dm]: To avoid people rigging the garbage bins, transporting them with a RCS, etc

Comment (TullyBurnalot) [reply code/game/objects/structures/crates_lockers/crates.dm]: ^
Unless you'd also like the full boobytrapping functionality to be added, @Fox-McCloud , this is really just to allow people to dump stuff into the can.

Comment (Fox-McCloud) [reply code/game/objects/structures/crates_lockers/crates.dm]: Even if that's the case, it's best to have a var that controls that behavior on the parent, then you set that var to `TRUE` or `FALSE` on children that you don't want to/want to have that behavior.

That said, I do think the rigging and stuff should be retained. Watching someone shock themselves from just opening a trash can would be hilarious.

**Comment by Fox-McCloud** [comment]
Lines 446-446 need to be moved *under* `iswrench`.

File: code/game/objects/structures/crates_lockers/crates.dm (hunk 2 above)
Comment (taukausanake) [reply code/game/objects/structures/crates_lockers/crates.dm]: Grayshirt: "I'm going to throw away my trash like a human being"
*opens trash can*
***BOOM***
Grayghost: "WHYYYYYYY"

**Pull request merged**